ci/build_docs.sh

```diff
@@ -69,6 +69,10 @@ if [ "$DOC" ]; then
            pandas/core/reshape/reshape.py \
            pandas/core/reshape/tile.py
 
+    echo "Running spellcheck on documentation"
```

Comment:
I tried to follow the example on line 35 where `make.py` is being called. If I'm not mistaken we need to cd into the doc folder before running the make.py spellcheck since the script cd back to `$TRAVIS_BUILD_DIR`.

This is the first time I tried to write something like this so I'd like your opinion about it and if enabling the spellchheck within CI should be done in a different way.

Thank you

Reply:
Why don't you just move this closer to line 35 so you don't need to keep changing directories?